components/clm/src/biogeochem/CNCarbonFluxType.F90

```diff
@@ -2008,15 +2008,6 @@ subroutine Summary(this, bounds, num_soilc, filter_soilc, num_soilp, filter_soil
        end do
 
 
-       ! total heterotrophic respiration (HR)
```

Comment:
it seems this is not a change from this pr; it is probably from some merge, just surprising it appears here.

Reply:
I think next is not up to date with master. I rebased the branch on master right before submitting the PR...

Reply:
My bad. PRs should be based on master anyways. The integrator will deal with the next/master shenanigans.